activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -62,7 +62,8 @@ class SQLite3Adapter < AbstractAdapter
         time:         { name: "time" },
         date:         { name: "date" },
         binary:       { name: "blob" },
-        boolean:      { name: "boolean" }
+        boolean:      { name: "boolean" },
+        bigint:       { name: "bigint" },
```

Comment:
The sqlite3 adapter has no `valid_type?` method. Why would we need to add this entry?